models/classes/QtiResultsService.php

```diff
@@ -67,7 +66,7 @@ public function getDeliveryExecutionByTestTakerAndDelivery($delivery, $testtaker
         $delivery = new \core_kernel_classes_Resource($delivery);
```

Comment:
Since other FQN's are being cleaned up, we can also change `\core_kernel_classes_Resource` to `core_kernel_classes_Resource` here for consistency